q2_types/feature_data/_format.py

```diff
@@ -100,34 +100,51 @@ class TSVTaxonomyFormat(model.TextFileFormat):
     """
     HEADER = ['Feature ID', 'Taxon']
 
-    def sniff(self):
+    def _check_tsv_tax_format(self, root, n=None):
```

Comment:
What does `root` do here? Looks like maybe this is copy-pasta from the manifest formats (?)

Reply:
`root` has now been removed from the function header. Apologies for the inclusion of that value in the function.